application/auth/views.py

```diff
@@ -84,3 +85,11 @@ def reset_password(token):
         return render_template("auth/password_updated.html", form=form, token=token, user=user)
 
     return render_template("auth/reset_password.html", form=form, token=token, user=user)
+
+
+@auth_blueprint.route("/logout", methods=["POST"])
+def logout():
+    session.clear()
+    from flask_security.views import logout as security_logout_view
+
+    return security_logout_view()
```

Comment:
Maybe we should explicitly redirect to the homepage here? I'm not sure what the `security_logout_view()` is?

Reply:
The security_logout_view is the thing that does the logging out.  It looks like there is an app config variable we can set, but the default behaviour is to redirect to root of the site, which I think is fine?

Reply:
👍